src/templates/docPage.js

```diff
@@ -78,6 +84,23 @@ const BasicDoc = ({ data, location, pageContext }) => {
   if (isTutorial) {
     type = 'tutorial';
   }
+  const useBannerDismissed = createLocalStorageStateHook(
+    `docBannerDismissed-${title}`
+  );
+
+  const [bannerDismissed, setBannerDismissed] = useBannerDismissed(null);
+  const [bannerVisible, setBannerVisible] = useState(bannerDismissed !== true);
```

Comment:
Do we check anywhere if they are logged in?

Reply:
😂 🙈 ohhoho not yet we don't

Reply:
fixed now!